plugins/de.cau.cs.kieler.klighd/src/de/cau/cs/kieler/klighd/KlighdDataManager.java

```diff
@@ -98,6 +98,9 @@
     
     /** name of the 'preservedProperties' element. */
     private static final String PRESERVED_PROPERTIES = "preservedProperties";
```

Comment:
I think the preserved properties are no longer in use now, correct?

Reply:
They're still used for copying some stuff from the elkgraph to the KGraph apparently.

https://github.com/kieler/KLighD/blob/37653b1a27f9cdfef12d376651d9b948d60d9b49/plugins/de.cau.cs.kieler.klighd.incremental/src/de/cau/cs/kieler/klighd/incremental/merge/KGraphMerger.java#L459

Reply:
No, I think you removed the only reference to it by introducing the blacklist and copying all other properties such as desiredPostion, layerId, etc...

Reply:
Again, is this still needed?

Reply:
Currently the only remaining reference is in the KGraphMerger. It's used there in conjunction with some property removal process to filter which properties should be copied from an elkgraph to a kgraph. This seems to be a separate thing from where preservedProperties have now been removed from and I'm not sure what the reasoning behind the filtering there is. To me it looks like preservedProperties were overloaded in their purpose before. I don't know what the right way to change KGraphMerger is. If that is solved, then yes preservedProperties and the related classes can be removed.